yt_dlp/extractor/drtv.py

```diff
@@ -123,22 +127,19 @@ class DRTVIE(InfoExtractor):
         'url': 'https://www.dr.dk/drtv/program/jagten_220924',
         'only_matching': True,
     }, {
-        'url': 'https://www.dr.dk/lyd/p4aarhus/regionale-nyheder-ar4/regionale-nyheder-2022-05-05-12-30-3',
+        'url': 'https://www.dr.dk/lyd/p4kbh/regionale-nyheder-kh4/regionale-nyheder-2023-03-14-10-30-9',
```

Comment:
add a new test instead of overwriting the old one

Reply:
The old one results in a 404. Should it still be kept but skipped instead?

Reply:
yeah you can add
```py
        'skip': 'this video has been removed',
```
like the first test has